### ISSUE TITLE
Bug: Most clickable elements are div, span or i elements

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#3209]
## Bug description & symptom

Most clickable elements in tracim are `<div>`, `<span>`, or `<i>` elements. This is a big accessibility issue, and prevents one to use the tab key to navigate Tracim.

`<a href>` or `<input type="button">` or `<button>` elements should be used instead.

Careful `tabindex` attributes could be used as a second step.

## Impacted components

- *frontend*

**Comment by yvan-algoo** [comment]
@lebouquetin
